src/components/Picklist/index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/role-has-required-aria-props */
```

Comment:
use in the line instead because this will not alert us if there is another error in the file